shell/platform/embedder/embedder.h

```diff
@@ -1791,9 +1916,9 @@ typedef struct {
   /// The callback will be invoked on the thread on which the `FlutterEngineRun`
   /// call is made.
   ///
-  /// @deprecated    Prefer using `update_semantics_callback` instead. If this
-  ///                calback is provided, `update_semantics_callback` must not
-  ///                be provided.
+  /// @deprecated    Use `update_semantics_callback2` instead. If this
+  ///                callback is provided, `update_semantics_callback2` and
+  ///                `update_semantics_callback` must not be provided.
```

Comment:
I'll admit it took me a couple reads to be sure I was parsing this right. What do you think of the following note for all three of these (with the notable exception that the latest callback isn't marked deprecated)?
```suggestion
  /// @deprecated    Use `update_semantics_callback2` instead. Only one of      
  ///                `update_semantics_node_callback`,                          
  ///                `update_semantics_callback`, and                           
  ///                `update_semantics_callback2` may be provided; the others
  ///                should be set to null. 
```

Reply:
Fixed